rollup.config.js

```diff
@@ -46,9 +46,14 @@ export default {
             browser: true
         }),
         commonjs({
-            // namedExports: {
-            //     'node_modules/@okta/okta-auth-js/dist/okta-auth-js.umd.js': ['toRelativeUrl']
-            // }
+            namedExports: {
+                // TODO: why is this necessary?
```

Comment:
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module

Reply:
Can we update this comment with an answer?

Reply:
This link explains why the error could be shown. I still don't understand why it is necessary. `okta-auth-js` is not a commonjs module. and these namedExports only seem to affect functions from the util file, other named exports are handled fine. So there is still a mystery here. When we figure it out, we should be able to remove this or explain it better.